app/builders/application_builder.rb

```diff
@@ -59,18 +61,21 @@ def dependent_attributes(online_application)
   end
 
   def online_applicant_attributes(online_application)
-    fields = %i[title first_name last_name date_of_birth ni_number married]
+    fields = [:title, :first_name, :last_name, :date_of_birth, :ni_number, :married]
```

Comment:
@leoapost did the new specification told you to do this or did you change the config for this? I thought that we agreed on the style before?